src/app/courses-page/courses-page/courses-page.component.html

```diff
@@ -1,12 +1,15 @@
 <div class="root">
   <div class="utility-area">
-    <app-search></app-search>
+    <app-search (search)="searchCourses($event)"></app-search>
     <app-new-course></app-new-course>
   </div>
-  <div class="main-area">
+  <div *ngIf="coursesToShow.length" class="main-area">
     <ul class="courses-list">
-      <app-course *ngFor="let course of courses" [course]="course" (delete)="deleteCourse($event)"></app-course>
+      <app-course *ngFor="let course of (coursesToShow | orderBy:'creationDate')" [course]="course" (delete)="deleteCourse($event)"></app-course>
     </ul>
     <button type="button" class="btn btn-outline-secondary" (click)="loadCourses()">LOAD MORE</button>
   </div>
+  <div *ngIf="!coursesToShow.length" class="no-data-message">
```

Comment:
it is better to use "ngIf else" here https://stackoverflow.com/questions/43006550/how-to-use-ngif-else